app/components/wallet/CreateVaultSteps/NewVault.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { vault } from '/assets/images';
+import { smColors } from '/vars';
+import Input from '../../../basicComponents/Input';
```

Comment:
we have absolute path definition in webpack, no need in '../../'

Reply:
fix